backends/occa/ceed-occa-operator.c

```diff
@@ -331,45 +331,52 @@ static int CeedOperatorApply_Occa(CeedOperator op,
   // Input Evecs and Restriction
   for (CeedInt i=0,iein=0; i<qf->numinputfields; i++) {
     dbg("\n[CeedOperator][Apply] %d/%d Input Evecs:",i,qf->numinputfields-1);
-    // Restriction
-    if (op->inputfields[i].Erestrict) {
+    // No Restriction
+    if (op->inputfields[i].Erestrict == CEED_RESTRICTION_IDENTITY) {
+      CeedEvalMode emode = qf->inputfields[i].emode;
+      if (emode & CEED_EVAL_WEIGHT) {
+        dbg("[CeedOperator][Apply] No restriction, WEIGHT");
+      } else {
+        // Active
```

Comment:
I also caught that the OCCA backend was not distinguishing between active and passive inputs with 'CEED_RESTRICTION_IDENTITY'.